lib/injection/data_module.dart

```diff
@@ -22,7 +22,7 @@ abstract class DataModule {
         userPreferences: injector(),
         refreshTokenHttpClient: injector(),
         onTokenExpired: () {
-          // TODO: Handle log out
+          // Handle log out
```

Comment:
Let's keep the todo here in order not to forget to handle refreshing token failure

Reply:
Dont know why these rolling PR's are not handling the merges better

Reply:
Yeah i just usually wait till 1 branch is merged instead of branching out of one, this only happens on github for some reason :(